packages/repack/js/NativeScriptManager.ts

```diff
@@ -0,0 +1,12 @@
+import type {TurboModule} from 'react-native';
+import {TurboModuleRegistry} from 'react-native';
+
+export interface Spec extends TurboModule {
+    readonly getConstants: () => {};
+
+    loadScript(scriptId: string, config: Object): Promise<Object>;
+    prefetchScript(scriptId: string, config: Object): Promise<Object>;
+    invalidateScripts(scripts: Array<string>): Promise<null>;
```

Comment:
Why promise null? Should this function return something? If not, it should not be async

Reply:
A Promise is needed on JS side to generate `resolve` and` reject `on objc and a `Promise` for kotlin which is included in the `invalidateScripts ` func, Although I skipped changing this to string. I will fix it now. Thanks.

Reply:
Fixed.